flow/util/template_filters.py

```diff
@@ -109,6 +109,61 @@ def calc_tasks(operations, name, parallel=False, allow_mixed=False):
         )
 
 
+def _parse_memory(memory):
```

Comment:
Rename this to make it clearer that it's parsing a string to a numerical value.
```suggestion
def _parse_memory_flag(memory):
```

Reply:
If this should be removed in 0.14 (when `--memory` is removed) then please add a deprecation notice or comment that contains the string "0.14" (that's what I'll search for when checking that deprecated code was removed).